core/controllers/dashboard.py

```diff
@@ -132,6 +133,9 @@ def get(self):
 class DashboardHandler(base.BaseHandler):
     """Provides data for the user's creator dashboard page."""
 
+    # Count num_plays for all versions
+    VERSION_ALL = 'all'
```

Comment:
Removed. Not needed now since num_plays is already included when fetched from summary_services.